### PR TITLE
Path updates

### PR DESCRIPTION
**Pull request by jkopriva**

### CODE REVIEW

**Comment by jkopriva** [comment]
Need to retest - please wait with merge

**Comment by jkopriva** [comment]
min install script is working

File: rhtap-cli/rhtap-full-install.sh [hunk 1]
```diff
@@ -81,7 +84,7 @@ install_rhtap() {
   githubapp_integration
   gitlab_integration  # you can comment it if you don't need it
   jenkins_integration # you can comment it if you don't need it
-  ./bin/rhtap-cli deploy --timeout 35m --config ./config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
+  ./bin/rhtap-cli deploy --timeout 35m --config ./installer/config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
```
Comment (xinredhat) [inline rhtap-cli/rhtap-full-install.sh]: I prefer defining a variable `readonly config_file="installer/config.yaml"`. If there is any change in future, we just need to change it in one place

File: rhtap-cli/rhtap-min-install.sh [hunk 2]
```diff
@@ -83,7 +87,7 @@ install_rhtap1() {
   jenkins_integration # you can comment it if you don't need it
   quay_integration
   acs_integration
-  ./bin/rhtap-cli deploy --timeout 35m --config ./config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
+  ./bin/rhtap-cli deploy --timeout 35m --config ./$config_file --kube-config "$KUBECONFIG" --debug --log-level=debug
```
Comment (xinredhat) [inline rhtap-cli/rhtap-min-install.sh]: removing `./` before `$config_file` , WDYT?

**Pull request merged**